services/cms/src/components/editors/GutenbergEditor.tsx

```diff
@@ -68,10 +68,10 @@ const GutenbergEditor: React.FC<GutenbergEditorProps> = ({
   >({})
 
   useEffect(() => {
-    if (courseId) {
-      setEditorSettings((prev) => ({ ...prev, mediaUpload: mediaUploadBuilder(courseId) }))
+    if (organizationId) {
+      setEditorSettings((prev) => ({ ...prev, mediaUpload: mediaUploadBuilder(organizationId) }))
```

Comment:
Pass the media upload function as a prop, the parent will configure it to either use the course id or the exam id (or the course instance id in the future)